server/grpc/newcore/proto.go

```diff
@@ -0,0 +1,100 @@
+package newcore
```

Comment:
Can you move the content of this file in `server/grpc/api/service.go`

Reply:
done